gold-cc-cvc-input.html

```diff
@@ -199,11 +199,10 @@
          */
         validate: function() {
           // Empty, non-required input is valid.
-          if (!this.required && this.value == '') {
-            return true;
-          }
-
-          var valid = this.value.length == this._requiredLength;
+          var valid =
+              (this.$.input.validate() &&
+              this.value.length == this._requiredLength) ||
+              !this.required && this.value == '';
```

Comment:
If we're assuming the `input` is an `iron-input` (which we are, since it has a `validate` method), then I don't think we need the required/value bits, since `iron-input` already checks those: https://github.com/PolymerElements/iron-input/blob/master/iron-input.html#L289